week-3/quote-generator/index.html

```diff
@@ -3,13 +3,22 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Quote generator app</title>
+    <link rel="stylesheet" href="style.css">
     <script defer src="quotes.js"></script>
   </head>
   <body>
-    <h1>hello there</h1>
+    <section class="layout">
+    <div class="content">
+    <h1>Hello there</h1>
     <p id="quote"></p>
+    </div>
+  <div class="side">
     <p id="author"></p>
     <button type="button" id="new-quote">New quote</button>
+    <input type="checkbox" id = "check-box"/>
+    <p id="auto-play-text"></p>
+  </div>
+</section>
```

Comment:
It's great to see that you're using semantic elements like `<section>`, `<h1>`,` <p>`, and `<button>`. 
As a further step, you can consider using a `<form>` and, you can improve the accessibility of your checkbox by adding a `<label>` element:
```
<form>
      <label for="check-box">Auto Play:</label>
      <input type="checkbox" id="check-box" />
 </form>
 ```